docs/function-format.md

```diff
@@ -58,42 +62,44 @@ The header keys and values would be populated with information about the functio
 
 Pros:
 
-* Streamable
+* Supports streaming
```

Comment:
our use of the word 'streamable' is confusing, simply because from a user perspective it's not really true. our JSON format is 1 object, which is not really streamable. i.e. we're going to read the entire request into a json object and then plop it over to the function input, we're not going to write each blob the user sends over into the container as we get it (i.e. as it's own json object or however we want to frame it). 

i know that internally we use this verbiage to delineate hot from cold, doesn't really make it any less confusing from a user perspective for us to expose this semantic alias. 'hot' and 'streamable' aren't interchangeable imo, we should say 'supports hot' and not 'streamable'.

Reply:
oh, this is the http section. true here, but not what was intended really ;)